[2024-10-03T05:23:48.628+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T05:23:48.654+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_preparation.split_data scheduled__2024-10-03T04:00:00+00:00 [queued]>
[2024-10-03T05:23:48.664+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_preparation.split_data scheduled__2024-10-03T04:00:00+00:00 [queued]>
[2024-10-03T05:23:48.664+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-03T05:23:48.678+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): split_data> on 2024-10-03 04:00:00+00:00
[2024-10-03T05:23:48.715+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3788) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-03T05:23:48.717+0000] {standard_task_runner.py:72} INFO - Started process 3807 to run task
[2024-10-03T05:23:48.727+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_preparation', 'split_data', 'scheduled__2024-10-03T04:00:00+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpst3gkbde']
[2024-10-03T05:23:48.743+0000] {standard_task_runner.py:105} INFO - Job 166: Subtask split_data
[2024-10-03T05:23:50.969+0000] {task_command.py:467} INFO - Running <TaskInstance: data_preparation.split_data scheduled__2024-10-03T04:00:00+00:00 [running]> on host 8e64ac78d004
[2024-10-03T05:23:51.127+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_preparation' AIRFLOW_CTX_TASK_ID='split_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-03T04:00:00+00:00'
[2024-10-03T05:23:51.131+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T05:23:51.491+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T05:23:51.553+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T05:23:51.555+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_preparation, task_id=split_data, run_id=scheduled__2024-10-03T04:00:00+00:00, execution_date=20241003T040000, start_date=20241003T052348, end_date=20241003T052351
[2024-10-03T05:23:51.679+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T05:23:51.740+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-03T05:23:51.743+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
